docs/source/developers/java/development.rst

```diff
@@ -0,0 +1,170 @@
+    [INFO] Arrow Memory - Netty ............................... SUCCESS [ 10.211 s]
+    [INFO] Arrow Vectors ...................................... SUCCESS [01:48 min]
+    [INFO] Arrow Compression .................................. SUCCESS [  7.282 s]
+    [INFO] Arrow Tools ........................................ SUCCESS [ 12.612 s]
+    [INFO] Arrow JDBC Adapter ................................. SUCCESS [ 15.558 s]
+    [INFO] Arrow Plasma Client ................................ SUCCESS [  6.303 s]
+    [INFO] Arrow Flight ....................................... SUCCESS [  2.326 s]
+    [INFO] Arrow Flight Core .................................. SUCCESS [ 50.632 s]
+    [INFO] Arrow Flight GRPC .................................. SUCCESS [  9.932 s]
+    [INFO] Arrow Flight SQL ................................... SUCCESS [ 22.559 s]
+    [INFO] Arrow Flight Integration Tests ..................... SUCCESS [  9.649 s]
+    [INFO] Arrow AVRO Adapter ................................. SUCCESS [ 17.145 s]
+    [INFO] Arrow Algorithms ................................... SUCCESS [ 44.571 s]
+    [INFO] Arrow Performance Benchmarks ....................... SUCCESS [  9.202 s]
+    [INFO] Arrow Java C Data Interface ........................ SUCCESS [ 15.794 s]
+    [INFO] Arrow Orc Adapter .................................. SUCCESS [ 14.205 s]
+    [INFO] Arrow Gandiva ...................................... SUCCESS [02:06 min]
+    [INFO] Arrow Java Dataset ................................. SUCCESS [ 15.261 s]
+    [INFO] ------------------------------------------------------------------------
+    [INFO] BUILD SUCCESS
+    [INFO] ------------------------------------------------------------------------
+
+Performance Testing
+===================
+
+Java module that contains performance tests is: `performance`
+
+Let's configure our performance environment to run performance tests:
+
+- Install `benchmark`_
+- Install `archery`_
+- Install `conbench`_
+
+Lets execute benchmark tests:
+
+.. code-block::
+
+    $ cd benchmarks
+    $ conbench java-micro --help
+    $ conbench java-micro
+        --iterations=1
+        --commit=e90472e35b40f58b17d408438bb8de1641bfe6ef
+        --java-home=<absolute path to your java home>
+        --src=<absolute path to your arrow project>
+        --benchmark-filter=org.apache.arrow.adapter.AvroAdapterBenchmarks.testAvroToArrow
+    Benchmark                              Mode  Cnt       Score   Error  Units
+    AvroAdapterBenchmarks.testAvroToArrow  avgt       725545.783          ns/op
+    Time to POST http://localhost:5000/api/login/ 0.14911699295043945
+    Time to POST http://localhost:5000/api/benchmarks/ 0.06116318702697754
+
+Then go to: http://127.0.0.1:5000/ to see reports:
+
+UI Home:
+
+.. image:: img/conbench_ui.png
+
+UI Runs:
+
+.. image:: img/conbench_runs.png
+
+UI Benchmark:
+
+.. image:: img/conbench_benchmark.png
+
+Code Style
+==========
+
+Code style are configured at `java/dev/checkstyle`
```

Comment:
Backticks don't do what you want in reStructuredText, use double-backticks

Reply:
Updated